File: bridge/c-sharp/IMessageSender.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Bridge
{
    // UML diyagramındaki Implementor (IMessageSender) arayüzünen denk gelmektedir.
    // Abstraction (Message) tarafından referansı tutulur ve haberleşme burası ile gerçekleşir.
    public interface IMessageSender
    {
        void SendMessage(Message message);
    }
}

File: bridge/kotlin/IMessageSender.kt
package Bridge

// UML diyagramındaki Implementor (IMessageSender) arayüzünen denk gelmektedir.
// Abstraction (Message) tarafından referansı tutulur ve haberleşme burası ile gerçekleşir.
interface IMessageSender {
    fun sendMessage(message: Message)
}
